deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -462,3 +462,51 @@ def check_error_when_updating_cve_database(log_monitor=None, timeout=vd.T_20):
     """
     check_vuln_detector_event(file_monitor=log_monitor, timeout=timeout,
                               callback=r"ERROR: .* CVE database could not be updated.")
+
+
+def check_cpe_helper_invalid_tag_warning(field='', log_monitor=None, timeout=vd.T_20):
+    """Check that a warning message comes out for invalid tags.
+
+    Args:
+        field (str): Tag name
+        log_monitor (FileMonitor): Log monitor.
+        timeout (str): timeout to check the event in Wazuh log.
+    """
+    check_vuln_detector_event(file_monitor=log_monitor, timeout=timeout,
+                              callback=fr"WARNING: .* Invalid tag found when parsing the CPE dictionary: '{field}'")
+
+
+def check_cpe_helper_invalid_tag_fail(log_monitor=None, timeout=vd.T_20):
+    """Check that a warning message comes out for invalid tags.
+
+    Args:
+        log_monitor (FileMonitor): Log monitor.
+        timeout (str): timeout to check the event in Wazuh log.
+    """
+    check_vuln_detector_event(file_monitor=log_monitor, timeout=timeout,
+                              callback=r"DEBUG: .* The 'Wazuh CPE Helper' update has failed, so the NVD feed will "
+                              "not be updated.")
+
+
+def check_cpe_helper_invalid_format_version(log_monitor=None, timeout=vd.T_20):
+    """Check that a error message comes out for invalid format version value.
+
+    Args:
+        log_monitor (FileMonitor): Log monitor.
+        timeout (str): timeout to check the event in Wazuh log.
+    """
+    check_vuln_detector_event(file_monitor=log_monitor, timeout=timeout,
+                              callback=r"ERROR: .* Invalid format version for the CPE helper: .*")
+
+
+def check_error_inserting_package(log_monitor=None, agent_id='000', timeout=vd.T_20):
+    """Check in the log that an error comes out when insert package with a missing tag in the cpe_helper.json file.
+
+    Args:
+        log_monitor (FileMonitor): Log monitor.
+        agent_id (str): Agent ID.
+        timeout (str): timeout to check the event in Wazuh log.
+    """
+    check_vuln_detector_event(file_monitor=log_monitor, timeout=timeout,
+                              callback=fr"ERROR: .* Could not insert the CPEs from the agent '{agent_id}' "
+                                       "into the database.")
```

Comment:
Raw string is not required
```suggestion
                              callback=f"ERROR: .* Could not insert the CPEs from the agent '{agent_id}' "
                                       "into the database.")
```

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3731/commits/044d5b1ff02734c0c25731bde55bd5c3831dc6be)